lib/handlers/allow.js

```diff
@@ -36,8 +37,12 @@ function allow (mode) {
       })
 
       // Ensure the user has the required permission
-      req.acl.can(req.session.userId, mode)
-             .then(() => next(), next)
+      const userId = req.session.userId
+      req.acl.can(userId, mode)
+        .then(() => next(), err => {
```

Comment:
Nitpick: can replace `() => next()` with `next`

Reply:
We can't: the promise resolves to a non-falsy value (`true`), and the first parameter of `next` being truthy is interpreted as an error.

Reply:
🤦‍♂️ 🤦‍♀️